### PR TITLE
LED rearrangement

### PR DESCRIPTION
**Pull request by CClarke513**

### CODE REVIEW

File: sketch_oct9a.ino [hunk 1]
```diff
@@ -278,7 +278,7 @@ void setup() {
   // Pin A6: analog input from VOC sensor
   pinMode(A6, PinMode::INPUT);
   // Pin D6: temp/humidity sensor LEDs
```
Comment (bbrk24) [inline sketch_oct9a.ino]: missed the comment here

**Review by bbrk24 — DISMISSED** [review]
You also missed the `digitalWrite` call

**Review by bbrk24 — COMMENTED** [review]
image

Works but I don't like how the wire overlaps the text. I think something like one of these would be better:

image image

**Review by DarthSethious1 — APPROVED** [review]
The file looks good to me.

**Comment by DarthSethious1** [comment]
Riley, whenever you're okay to merge, go ahead and do it

**Pull request merged**